en/mtls.md

```diff
@@ -0,0 +1,283 @@
+---
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+title: "Securing Vespa with mutually authenticated TLS (mTLS)"
+---
+
+## Introduction
+{% include note.html content="See accompanying [reference](reference/mtls.html) for details on for instance configuration syntax." %}
+{% include note.html content="This document is only relevant if you're **self-hosting Vespa**.
+If you're using Vespa Cloud all services are automatically set up securely by default with full mTLS,
+with all key/certificate management handled for you.
+" %}
+{% include note.html content="Though the terms *TLS* and *mTLS* may be used interchangeably in this document, *TLS* implies *mTLS* for all cluster-internal traffic." %}
```

Comment:
As discussed offline, these notes should probably be converted into paragraphs inline with the text itself